### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

### CODE REVIEW

**Comment by gunjanmimo** [comment]
@leecalcote sir, may you check these changes? I am not able to see all the check in action for this change?

**Comment by leecalcote** [comment]
> @leecalcote sir, may you check these changes? I am not able to see all the check in action for this change?

Weird. I don't see any actions being invoked here either. Just the DCO check. I'll kick off the workflow again now...

**Comment by leecalcote** [comment]
Screen Shot 2020-02-17 at 8 11 24 AM

**Comment by leecalcote** [comment]
@gunjanmimo there was just a bit of massaging needed in the spacing and line returns in the ci.yml. We should be good now.

File: linkerd/yaml.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (d *YAMLDecoder) Close() error {
 }
 
 const yamlSeparator = "\n---"
-const separator = "---"
```
Comment (nupur-thakur) [inline linkerd/yaml.go]: @gunjanmimo can you explain what exactly is this change is doing and why it is required?

Comment (gunjanmimo) [reply linkerd/yaml.go]: this "_" I used to avoid the error message "declared bu not used"

Comment (nupur-thakur) [reply linkerd/yaml.go]: Okay @gunjanmimo Thanks, Got it.

Comment (gunjanmimo) [reply linkerd/yaml.go]: @nupur-thakur do you know, why git action is not showing me all the checks?

**Pull request closed without merging**